TreeTracker/UI/Views/View Trees/ViewTreesViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewTreesViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 10/27/20.
+//  Copyright © 2020 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ViewTreesViewController: UIViewController {
+    var viewModel: ViewTreesViewModel? {
+        didSet {
+            viewModel?.viewDelegate = self
+            title = viewModel?.title
+        }
+    }
+   @IBOutlet weak var viewTrees: UICollectionView!
+   //private let treeMonitoringService: TreeMonitoringService
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+         viewTrees.delegate = self
+         viewTrees.dataSource = self
+         viewTrees.register(ViewTreesCollectionViewCell.self, forCellWithReuseIdentifier: "ViewTreesCollectionViewCell")
+         viewModel?.fetchTrees()
+
+    }
+}
+
+extension ViewTreesViewController: UICollectionViewDelegate, UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 5
```

Comment:
This still needs to be populated via the presenter im guessing?

Reply:
ok